<launch>

<node pkg="base_estimation"
        type="base_estimation_node"
        name="cogimon_base_estimation"
        output="screen">

    <param name="ik_problem" textfile="$(find base_estimation)/config/cogimon_base_estimation.stack"/>
    <param name="rate" value="200.0"/>
    <param name="surface_contacts/l_sole" value="l_sole"/>
    <param name="surface_contacts/r_sole" value="r_sole"/>
    <param name="contact_attach_thr" value="100.0"/>
    <param name="contact_release_thr" value="50.0"/>
    <param name="velocity_filter/omega" value="100.0"/>
    <param name="contact_release_thr" value="50.0"/>
    <param name="use_imu" value="true"/>

    <remap from="/robot_description" to="/xbotcore/robot_description"/>
    <remap from="/robot_description_semantic" to="/xbotcore/robot_description_semantic"/>

</node>

</launch>
